### DISCUSSION TITLE
Forwarding entertainment packets directly to Zigbee (Hue) lights?

### DISCUSSION THREAD

**Discussion by sofakng** [hass-emulated-hue/core#195] [General]
I'm currently using a Hue bridge with Hue lights and the entertainment mode is really impressive however I'd love to add some non-Hue lights (ie. Tasmota/ESPHome or WLED lights) but of course it isn't possible on the official Hue bridge.

I was looking for Hue hub emulators and found diyHue but their entertainment mode is supposedly slow so I was excited to find this project.

It looks like forwarding entertainment UDP packages to Zigbee mesh for Official HUE lights is on the roadmap but I'm wondering if there has been any progress or if this is still being planned?

**Comment by alexyao2015** [comment]
This isn't something being planned. If anything were to be done, it would be through a custom component in esphome as with #85. However, that haven't been implemented simply because that would vastly increase the complexity of the codebase.
